src/Board.js

```diff
@@ -0,0 +1,33 @@
+function Square({ value, onClick }) {
```

Comment:
In terms of project structure, you should keep each component in its own file. I will recommend you move the `Square` component to a different file and import it to `Board`.